File: python/fizz_buzz.py

def buzz_fizz(upper_bound,lower_bound,arg1,arg2):
	for i in range(lower_bound,upper_bound+1):
		print("FizzBuzz" if ((i%arg1==0)and(i%arg2==0)) else ("Fizz" if (i%arg1==0) else ("Buzz" if (i%arg2==0) else i)))


#driver code
print("enter upper bound,lower bound, arg1 ,arg2\n arg are the numbers of which multiples will be checked")
upp=int(input())
lower=int(input())
a1=int(input())
a2=int(input())
buzz_fizz(upp,lower,a1,a2)

File: cpp/fizz_buzz.cpp
#include <iostream>
// working....
void buzzFizz(int upper_bound,int lower_bound,int arg1,int arg2){
	for (int i = lower_bound; i <=upper_bound; i++){
		if((i%arg1==0)&&(i%arg2==0)){
			std::cout<<"FizzBuzz"<<std::endl;
		}
		else if(i%arg1==0){std::cout<<"Fizz"<<std::endl;}
		else if(i%arg2==0){std::cout<<"Buzz"<<std::endl;}
		else{std::cout<<i<<std::endl;}
	}
}
int main(){
	std::cout<<"enter upper bound,lower bound, arg1 ,arg2\n arg are the numbers of which multiples will be checked"<<std::endl;
	int upp,lower,a1,a2;
	std::cin>>upp;
	std::cin>>lower;
	std::cin>>a1;
	std::cin>>a2;
	buzzFizz(upp,lower,a1,a2);
}